docs/basics/faq-troubleshooting.md

```diff
@@ -424,3 +424,8 @@ can match from multiple sources. `cross-seed` is simply offering you the ability
 to revive the stalled torrents inside your client.
 
 :::
+
+### Why do I have so many torrents that are not being searched, a.k.a not "suitable to search for matches"? 
+
+Most likely this is expected behavior due to `config.js` settings such as `includeSingleEpisodes`,
+`includeNonVideos`. Check verbose log in app config folder for `[prefilter]` entries.
```

Comment:
```suggestion
### Why are some torrents not suitable for searching?

Seeing a difference in total torrent count and suitable torrents (as well as
unique searches) is expected behavior. This is usually due to `config.js`
settings such as `includeSingleEpisodes`or `includeNonVideos`, `exclude` options
based on age or last date of search, or duplicate search queries which have been
reduced down to a single query.

:::tip

You can check the verbose log in the log directory of the app config folder for
`[prefilter]` entries immediately before a search starts for exact details.

:::
```

I know this is a pretty dramatic elaboration/rework, but let me know what you think and we can work from here. 

Also, not entirely sure placing this at the very end is where I'd put it, but we can worry about the placement later (I try and keep super common things more near the top) - and this seems more common than the very end. But I understand why you placed it there lol.